src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx

```diff
@@ -111,6 +111,14 @@ export default function createQueryBuilder(initial = {}, organization) {
     // Default to all projects if none is selected
     const projects = query.projects.length ? query.projects : defaultProjects;
 
+    // Default to DEFAULT_STATS_PERIOD when no date range selected (either relative or absolute)
+    const {range, start, end} = query;
+    const hasAbsolute = start && end;
+    const daterange = {
+      ...(hasAbsolute && {start, end}),
+      ...(range || (!hasAbsolute && {range: DEFAULT_STATS_PERIOD})),
+    };
+
```

Comment:
I think I don't follow why we'd need to change this `getExternal` function since both the internal format and the format of the API which this function converts to shouldn't have changed in any way. The default stats period is already always applied by Discover if none is provided.